tests/client/amo/test_utils.js

```diff
@@ -1,29 +1,81 @@
 import createStore from 'amo/store';
+import * as featuredActions from 'amo/actions/featured';
 import * as landingActions from 'amo/actions/landing';
 import * as api from 'core/api';
 import {
+  ADDON_TYPE_EXTENSION,
   ADDON_TYPE_THEME,
   SEARCH_SORT_POPULAR,
   SEARCH_SORT_TOP_RATED,
 } from 'core/constants';
-import { loadLandingAddons } from 'amo/utils';
+import {
+  fetchFeaturedAddons,
+  loadFeaturedAddons,
+  loadLandingAddons,
+} from 'amo/utils';
 
 
 describe('amo/utils', () => {
-  describe('loadLandingAddons()', () => {
-    const addonType = ADDON_TYPE_THEME;
-    let ownProps;
-
-    beforeEach(() => {
-      ownProps = {
-        params: {
-          application: 'android',
-          visibleAddonType: 'themes',
-        },
-      };
+  let ownProps;
+
+  beforeEach(() => {
+    ownProps = {
+      params: {
+        application: 'android',
+        visibleAddonType: 'extensions',
+      },
+    };
+  });
+
+  describe('fetchFeaturedAddons()', () => {
```

Comment:
This test is exactly the same as `loadFeaturedAddons()`. I think it would make your life way easier if you merge the code for `fetchFeaturedAddons` into `loadFeaturedAddons` and only keep the test for `loadFeaturedAddons`. I know you mentioned that it follows a convention but it doesn't seem helpful in this context.